### DISCUSSION TITLE
What are the pros and cons of using APISIX as an Ingress Controller? Does APISIX have more advantages than Trafik in high traffic scenarios?

### DISCUSSION THREAD

**Discussion by ghost** [apache/apisix#7161] [Q&A]
What are the pros and cons of using APISIX as an Ingress Controller? Does APISIX have more advantages than Trafik in high traffic scenarios?

**Comment by tao12345666333** [comment]
You can see the comparison between the various Ingress controllers in the following documentation
https://docs.google.com/spreadsheets/d/191WWNpjJ2za6-nbG4ZoUMXMpUK8KlCIosvQB0f-oq3k/edit#gid=907731238

In addition to functional advantages, there are also architectural advantages: Apache APISIX Ingress controller is architecturally divided into a control plane and a data plane, and you can deploy them separately, either within a Kubernetes cluster or outside of a Kubernetes cluster.

Apache APISIX is the actual data plane, its performance is very high, and it is advantageous in large traffic scenarios
